vue.config.js

```diff
@@ -1,8 +1,7 @@
 module.exports = {
```

Comment:
I will have to check, but I think can we probably pull the needed values here (path, etc.) directly from the relevant .env files.  Do you think that would be simpler, so that we have one place to put settings (the .env files) and those settings get picked wherever needed, like here in vue.config.js?  Or do you prefer the current setup?  I am happy to do what you find most intuitive.